graylog2-web-interface/src/components/streams/StreamForm.jsx

```diff
@@ -84,14 +85,15 @@ class StreamForm extends React.Component {
     this.setState(change);
   };
 
-  state = this._getValuesFromProps(this.props);
+  _indexSetSelect = () => {
+    const { indexSetId } = this.state;
 
-  render() {
-    const { title, description, removeMatchesFromDefaultStream, indexSetId } = this.state;
+    if (!AppConfig.isCloud()) {
```

Comment:
It should be the other way around, or? :)

```suggestion
    if (AppConfig.isCloud()) {
```

Reply:
Hah! I used that to test the opposite. Good catch!